good-enough-practices-for-scientific-computing.tex

```diff
@@ -807,11 +807,11 @@ \section{Version Control}\label{sec:versioning}
   assistance for merging simultaneous changes.
 
 \item
-  \recommend{Create, maintain, and use a checklist for committing and
+  \recommend{Create, maintain, and use a checklist for logging and
```

Comment:
Mm - the word "logging" has a specific technical meaning to me that's different from "committing", but that may not be a problem for our audience. Can others please chime in?